### DISCUSSION TITLE
Animated expressions

### DISCUSSION THREAD

**Discussion by ghost** [element-hq/element-meta#802] [Ideas]
### Have you considered any alternatives?

Animated emojis next to the message

![SmartSelect_20221102-172719_Telegram](https://user-images.githubusercontent.com/104249296/199501665-40c92c32-ca1e-4b8e-94a8-58bde52055a6.gif)

### Are you willing to provide a PR?

No

**Comment by kittykat** [comment]
@Nizami20052022 Do you mean that you'd like the emoji to be a moving animation?

**Reply by ghost** [reply]
Yes
